docs/dev/your_first_pr.md

```diff
@@ -0,0 +1,251 @@
+# Your First PR!
+
+Thanks for choosing to help improve the Tanzu Framework! This guide will
+help you build your first successful pull request.
+
+# Opening an Issue
+
+Your new idea starts with a new GitHub issue. You'll link your pull request to
+your issue once you're ready to submit it. This will help us appropriately
+prioritize the release into which your PR should get merged into.
+
+[Click here](https://github.com/vmware-tanzu/tanzu-framework/issues/new) to
```

Comment:
Could we get this updated to use `https://github.com/vmware-tanzu/tanzu-framework/issues/new/choose` instead so it goes through our templates?